Source/sdc/sdc_newton_solve.H

```diff
@@ -76,107 +69,52 @@ f_sdc_jac(const Real dt_m,
         eos_state.xn[n] = U_full[UFS+n] / U_full[URHO];
     }
 #if NAUX_NET > 0
-    for (int n = 0; n < NumAux; ++n) {
-        eos_state.aux[n] = U_full[UFX+n] / U_full[URHO];
-    }
+    amrex::Error("error: aux data not currently supported in true SDC");
```

Comment:
What is the reason for this?

Reply:
I'm lazy and don't want to test whether it actually works right now, since this PR was already a lot of work.